src/filters/body.rs

```diff
@@ -325,6 +325,13 @@ impl fmt::Display for BodyDeserializeError {
 
 impl StdError for BodyDeserializeError {}
 
+impl BodyDeserializeError {
+    /// Access the cause of the deserialization error
+    pub fn cause(&self) -> &BoxError {
```

Comment:
I'd make this the `source()` method in the `impl StdError`.

Reply:
Yep, that makes more sense. fixed!